src/app/public/modules/ag-grid/cell-editors/cell-editor-autocomplete/cell-editor-autocomplete.component.spec.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  async,
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  Column,
+  ICellEditorParams,
+  RowNode
+} from 'ag-grid-community';
+
+import {
+  SkyAgGridCellEditorAutocompleteComponent
+} from '../cell-editor-autocomplete';
+
+describe('SkyCellEditorAutocompleteComponent', () => {
+  let fixture: ComponentFixture<SkyAgGridCellEditorAutocompleteComponent>;
+  let component: SkyAgGridCellEditorAutocompleteComponent;
+  let nativeElement: HTMLElement;
+
+  const data = [
+    { id: '1', name: 'John Doe', town: 'Daniel Island' },
+    { id: '2', name: 'Jane Doe', town: 'Daniel Island' },
+    { id: '3', name: 'John Smith', town: 'West Ashley' },
+    { id: '4', name: 'Jane Smith', town: 'Mt Pleasant' }
+  ];
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SkyAppTestModule
+      ]
+    });
+
+    fixture = TestBed.createComponent(SkyAgGridCellEditorAutocompleteComponent);
+    nativeElement = fixture.nativeElement;
+    component = fixture.componentInstance;
+  });
+
+  it('renders an autocomplete input', () => {
+    fixture.detectChanges();
+
+    let element = nativeElement.querySelector('sky-autocomplete');
+    expect(element).toBeVisible();
+  });
+
+  describe('#agInit', () => {
+    let cellEditorParams: ICellEditorParams;
+    let column: Column;
+    const columnWidth = 200;
+    const selection = data[0];
+    const rowNode = new RowNode();
+    rowNode.rowHeight = 37;
+
+    beforeEach(() => {
+      column = new Column(
+        {
+          colId: 'col'
+        },
+        undefined,
+        'col',
+        true);
+
+      column.setActualWidth(columnWidth);
+
+      cellEditorParams = {
+        value: selection,
+        column,
+        node: rowNode,
+        keyPress: undefined,
+        charPress: undefined,
+        colDef: {},
+        columnApi: undefined,
+        data: undefined,
+        rowIndex: undefined,
+        api: undefined,
+        cellStartedEdit: undefined,
+        onKeyDown: undefined,
+        context: undefined,
+        $scope: undefined,
+        stopEditing: undefined,
+        eGridCell: undefined,
+        parseValue: undefined,
+        formatValue: undefined
+      };
+    });
+
+    it('initializes the SkyuxNumericCellEditorComponent properties', () => {
```

Comment:
Same as previous comment about using "should." Apply this to the rest of the tests.